### PR TITLE
feature: added first product page

### PR DESCRIPTION
**Pull request by poojakelkar**
- Added First Product Page

Live Link: https://neog-e-comm-website.netlify.app/firstproduct/

### CODE REVIEW

File: firstProduct/firstproductstyles.css [hunk 1]
```diff
@@ -1,140 +1,128 @@
...
 }
-.wishlist {
-    margin: 40px;
-    font-size: 32px;
-    font-weight: 700;
+.card .fa-lg {
+    position: absolute;
+    top: 1rem;
+    right: 0.1rem;
+    color: var(--heart-icon-color);
```
Comment (HrishikeshMahalle) [inline firstProduct/firstproductstyles.css:68]: for value that small you can use px

**Review by HrishikeshMahalle — APPROVED** [review]
Looks Good to merge. Updated code looks amazing.

File: firstProduct/firstproductstyles.css (hunk 1 above)
Comment (poojakelkar) [reply firstProduct/firstproductstyles.css:68]: Ok thanks

**Review by pratik-kumar-sahu — APPROVED** [review]
- Get rid of commented codes in final merge.
- UI looks clean. Looks good to merge.

**Comment by poojakelkar** [comment]
Thanks @pratik-kumar-sahu

**Pull request merged**